include/test/battle.h

```diff
@@ -483,8 +500,9 @@
 #define BATTLE_TEST_STACK_SIZE 1024
 #define MAX_TURNS 16
 #define MAX_QUEUED_EVENTS 25
+#define MAX_EXPECTED_ACTIONS 10
 
-enum { BATTLE_TEST_SINGLES, BATTLE_TEST_DOUBLES, BATTLE_TEST_WILD };
+enum { BATTLE_TEST_SINGLES, BATTLE_TEST_DOUBLES, BATTLE_TEST_WILD, BATTLE_TEST_AI, BATTLE_TEST_AI_DOUBLES };
```

Comment:
I'd call these `BATTLE_TEST_AI_SINGLES` and `BATTLE_TEST_AI_DOUBLES` with consistency with `BATTLE_TEST_SINGLES` and `BATTLE_TEST_DOUBLES`.